sport/app/football/controllers/MatchListController.scala

```diff
@@ -31,10 +32,14 @@ trait MatchListController extends BaseController with Requests {
   )(implicit request: RequestHeader, context: ApplicationContext) = {
     Cached(10) {
       if (request.isJson && request.forceDCR) {
+        val pageType: PageType = PageType(page, request, context)
         val model = DotcomRenderingFootballDataModel(
           matchesList = DotcomRenderingFootballDataModel.getMatchesList(matchesList.matchesGroupedByDateAndCompetition),
```

Comment:
Given that you now have an `apply` method, would it make sense to pass `matchesGroupedByDateAndCompetition` to that, and have it call a (`private`) `getMatchesList` method to build the `matchesList`?

Reply:
Good suggestion, I'll do this tomorrow and make a new PR 👍